source/automation.txt

```diff
@@ -6,48 +6,120 @@ Automation
-
-These examples also apply other recommended configurations, including:
-
-.. tabs::
-
-   .. tab:: Dev and Test Environments
-      :tabid: devtest
-
-      .. include:: /includes/shared-settings-clusters-devtest.rst
-
-   .. tab:: Staging and Prod Environments
-      :tabid: stagingprod
-
-      .. include:: /includes/shared-settings-clusters-stagingprod.rst
-
-.. tabs::
-
-   .. tab:: CLI
-      :tabid: cli
-
-      Content here
-
-   .. tab:: Terraform
-      :tabid: Terraform
-
-      Content here
+You can automate the management, provisioning, and configuration of
+|service| building blocks like database users and roles, |service|
+{+clusters+}, projects and organizations. Additionally, you can automate 
+setting up and managing of additional {+clusters+} resources, including
+enabling auto-scaling and disk scaling, creating and updating
+multi-cloud {+clusters+}, monitoring {+cluster+} performance and health,
+automating backups and restores, defining backup policies, and many more.   
+
+You can align your choice of tools with your preferred workflow to
+ensure seamless integration of |service-fullname| into your existing
+processes. The following |service-fullname| tools allow you to easily
+deploy and manage |service| at scale with repeatable, accurate, and
+scalable processes. 
+
+{+atlas-admin-api+} 
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The {+atlas-admin-api+} provides a |rest|\ful interface that allows you
+to leverage your preferred client such as cURL, Postman, and so on to
+directly interact with |api| endpoints that correspond to |service|
+resources.   
+
+To learn more, see :ref:`atlas-admin-api-access`.
+
+GoSDK Client 
+~~~~~~~~~~~~
+
+The GoSDK client simplifies interaction with the {+atlas-admin-api+} by
+providing pre-built functions and full |api| endpoint coverage. The SDK
+provides platform-specific and GoLang language-specific tools,
+libraries, and documentation to help build applications quickly and
+easily.  
+
+To learn more, see :ref:`Atlas Go SDK <atlas-admin-sdk>`.
+
+Infrastructure as Code (IaC) Integrations 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The integrations with the following :abbr:`IaC (Infrastructure as Code)`
+tools automate the deployment of |service| resources.
+
+.. list-table:: 
+   :stub-columns: 1 
+
+   * - HashiCorp Terraform MongoDB Atlas Provider
+     - Provisions |service| resources across cloud providers (|aws|,
+       |azure|, |gcp|) in the workflow of your choice. It allows you to
+       integrate |service| into your continuous delivery workflows with 
+       the official plugin that is verified and tested by HashiCorp.
+
+       Alternatively, you can use the :abbr:`CDKTF (Cloud Development
+       Kit for Terraform)` to deploy |service| in preferred languages
+       such as JavaScript, TypeScript, Python, Java, C#, and Go. 
+
+       To learn more, see :ref:`getting-started-terraform` and
+       :ref:`Deploy MongoDB Atlas with Terraform
+       <getting-started-terraform>`.  
```

Comment:
These links go to the same place, so changing one to the Hashicorp link. Also adding that Terraform examples are provided throughout these docs.

```suggestion
       To learn more, see :ref:`getting-started-terraform` and |service-terraform|. You can also use the Terraform examples in the {+atlas-arch-center+} such as the :ref:`Org, Project, and Cluster<arch-center-hierarchy>` examples to get started.
```